thyrel-web/src/components/room/lobby/StartButton.tsx

```diff
@@ -21,8 +26,9 @@ export default function StartButton({ player, onStart }: StartButtonProps) {
         )}
         <Typography variant="body2" color="textSecondary">
           {player?.isOwner
-            ? "you're the owner, click here to start the game"
-            : 'Waiting for the host to start the game.'}
+            ? "you're the owner, click here to start the " +
+              (book ? 'book.' : 'game.')
```

Comment:
cest bien mais ce serait encore mieux un truc + modulable
```suggestion
            ? `you're the owner, click here to start the ${startName}.`
```
avec une props "start name" qui est une string